pyproject.toml

```diff
@@ -25,7 +25,7 @@ packages = [{ include = "pineko", from = "src" }]
 [tool.poetry.dependencies]
 python = ">=3.8,<3.11"
 eko = "^0.12.0"
-pineappl = "^0.5.9"
+pineappl = "0.6.0a1"
```

Comment:
I believe you have to allow prereleases here https://python-poetry.org/docs/dependency-specification/#expanded-dependency-specification-syntax

Reply:
Ok thank you!